app/src/main/res/layout/activity_auth.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ViewSwitcher xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/view_switcher"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".AuthActivity">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/loading_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ProgressBar
+            android:id="@+id/loading_spinner"
+            style="?android:attr/progressBarStyle"
+            android:layout_width="match_parent"
+            android:layout_height="150dp"
+            android:gravity="center"
+            android:textAlignment="center"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <LinearLayout
+        android:id="@+id/verify_email_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center"
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/verify_email_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="40dp"
+            android:layout_marginBottom="10dp"
+            android:gravity="center"
+            android:text="@string/verify_email_text"
+            android:textSize="12pt" />
+
+        <Button
```

Comment:
If possible, it would be good to have a refresh button or something so that when the account is verified via email link on another device, you could refresh the app to let you through this page. For example, I clicked the email link on my computer and didn't know what to do on the phone because it didn't automatically let me through.

Reply:
Also, not entirely related, but is it possible to do a "remember me" option?

Reply:
It should refresh automatically every 5 seconds. It didn't work for you?

Reply:
Oh, you're right, it does. Maybe add text telling the user to allow for a few seconds of delay?

Reply:
Maybe I can just increase the refresh rate to be every 3 seconds? Or 2 seconds.

Reply:
If it doesn't have a noticeable battery penalty then yes